### PR TITLE
Create Publish to Template email instance and method.

### PR DESCRIPTION
**Pull request by billfitzgerald0120**
Created a new instance and method for Publish to Template for Infrastructure.
Modified State machine to use new instance.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1698542

@miq-bot add_label bug, hammer/yes
@miq-bot assign @tinaafitz

### CODE REVIEW

File: content/automate/ManageIQ/System/Notification/Email.class/__methods__/miq_provision_template_customize_body.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                body += "set retirement date."
+              end
+              body += " As the designated owner you will receive expiration warnings at this email address: #{to}"
+              body += "<br/><br/> If you have any issues with your new template please contact Support."
+              body += "<br/><br/> Thank you,"
+              body += "<br/> #{signature}"
+              @handle.object['body'] = body
+            end
+
+            def prov
```
Comment (lfu) [inline content/automate/ManageIQ/System/Notification/Email.class/__methods__/miq_provision_template_customize_body.rb]: Please use a full word as the method name.

**Comment by billfitzgerald0120** [comment]
@lfu I made changes as you requested, please review.

**Comment by lfu** [comment]
@billfitzgerald0120 Have you pushed up your changes?

**Comment by lfu** [comment]
LGTM

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit 2362c535bd1b293bcf8885f9fd2a3184f0d3062a
Author: tina <tfitzger@redhat.com>
Date:   Fri May 10 14:10:43 2019 -0400

    Merge pull request #529 from billfitzgerald0120/email_infra_template_complete2
    
    Create Publish to Template email instance and method.
    
    (cherry picked from commit b7bd3166ef3678d49b443298ff14d3e14c9bfb35)
    
    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1747186
```

**Pull request merged**